### ISSUE TITLE
Mysql data mirroring Issue

### ISSUE THREAD

**Issue by animer3009** [airbytehq/airbyte#13661]
- **Airbyte version**:  0.39.16-alpha
- **OS Version / Instance**: GCP Ubuntu 22.04 LTS
- **Deployment**: Docker
- **Source Connector and version**: Mysql 0.5.11
- **Destination Connector and version**: Mysql 0.5.11
- **Step where error happened**:  Sync job 

I have two Mariadb databases. I need to sync some tables from source to destination.
As I do not find a connector for Mariadb, I am using Mysql one. I am not sure if it can cause an issue.
The Problem happens when I am trying to mirror my table from source to destination.
Process finish successfully, but When I am checking destination database:

1) Table includes additional columns.
2) Schema includes additional tables like _airbyte_raw_.
3) Table column data types and order are not correct.
4) Data is not clear. For example my table has rows with combined identifiers and json string and the destination side I am getting strange changed rows...

For example source table has records (columns: key (type varchar), value (type text)):
'MYSQL_MIGRATION_LEVEL', '1'
'globalAuthor:a.ErjpNO9Pd6wx5RRu', '{"colorId":51,"name":null,"timestamp":1652452016283}'

Destination has  (columns: key (type text), value (type text)):
'TVlTUUxfTUlHUkFUSU9OX0xFVkVM', '1'
'Z2xvYmFsQXV0aG9yOmEuRXJqcE5POVBkNnd4NVJSdQ==', '{"colorId":51,"name":null,"timestamp":1652452016283}'

I tried different tables.
Everyware it is adding additional columns:
_airbyte_ab_id
_airbyte_emitted_at
_airbyte_normalized_at
_airbyte_copy_store_hashid
Also adding raw table like _airbyte_raw_
Not sure if these additional ones are OK to be there...

Almost everywhere I have issues with destination table types (for example source table has type int and destination gets float). There Is also a column order issue.

**Comment by alafanechere** [comment]
Hey @animer3009 this is not a bug this is how Airbyte works 😄 
You will find explanation on the purpose of these tables [here](https://docs.airbyte.com/understanding-airbyte/basic-normalization/).
In a nutshell, these are tables created by Airbyte with additional valuable metadata. We encourage users to build the final table or views from these raw or normalized tables if they want to get rid of metadata columns.
If you need further assistance please post on our [forum](https://discuss.airbyte.io/) for this kind of question.

**Comment by animer3009** [comment]
Hi @alafanechere 
So you mean change original column value is not a bug?
Are you sure lol.
About additional I may agree except columns order change and cast types from varchar to text and from integer to float..

**Comment by alafanechere** [comment]
Let me try to explain it in a different way 😄 
Airbyte is not mirroring 1:1 data from a source to a destination because the common use case Airbyte support is to replicate data from a source (API/ Database) to a different destination system. Because of the heterogeneity of these systems, it's not possible to exactly replicate schemas and types.
This is why Airbyte keeps the **raw data** it extracted from a source as JSON blobs stored in the destination and it internally stores the schema of this data with optional metadata to enrich the native and limited JSON types.
After replicating the raw data Airbyte can optionally run a normalization process. This is a best-effort, automatically generated DBT project which is reading the JSON blob and trying to create a normalized table on the destination according to the JSON schema it stored from the source. The normalization process can be opinionated and tries to cover the most common use cases. It is also limited by the features of the destination. And finally, it's still improvable, casting an integer to float is not right (we are working on it).
If you are not happy with the current normalization you can still use [custom DBT transformation](https://docs.airbyte.com/operator-guides/transformation-and-normalization/transformations-with-airbyte/) and run them from Airbyte to recreate the destination schema you expect.

**Comment by animer3009** [comment]
@alafanechere I got this, But once again:
It is changing also original column values.
For example original source column values was:
'MYSQL_MIGRATION_LEVEL'
and
'globalAuthor:a.ErjpNO9Pd6wx5RRu'
 
 In normalized table I got:
 'TVlTUUxfTUlHUkFUSU9OX0xFVkVM'
 and
 'Z2xvYmFsQXV0aG9yOmEuRXJqcE5POVBkNnd4NVJSdQ=='
 
I am sure this is not a normal.
 I was facing this issue while I not changed destination schema name (before was same as source).
 And also Destination Namespace to destination default.
 Now it is ok.
 
 Also column order: 
 For example source table column order (col1 col2 col3) in some cases after normalization changing order (col2 col1 col3 or possible different combinations too)...
 I guess this is not normal too...
